kong/client.go

```diff
@@ -272,6 +278,16 @@ func (c *Client) Do(ctx context.Context, req *http.Request,
 	return response, err
 }
 
+func ErrorOrResponseError(res *Response, err error) error {
```

Comment:
Godoc 🙏 ? At the end of the day this is exported so anyone from outside can also see it and use it

Reply:
done